src/pages/GivePage/EpochStatementDrawer.tsx

```diff
@@ -200,6 +200,8 @@ export const EpochStatementDrawer = ({
           value={statement}
           onChange={e => statementChanged(e.target.value)}
           placeholder="Summarize your Contributions"
+          // eslint-disable-next-line jsx-a11y/no-autofocus
```

Comment:
? what's this for?

Reply:
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-autofocus.md

Reply:
hmm ok

Reply:
This is to ignore the accessibility warning about hijacking focus.  in this case we are returning the user's focus to the drawer trigger they clicked, which is the ideal a11y behavior